models/userSchema.js

```diff
@@ -0,0 +1,66 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcryptjs');
+const validator = require('validator');
+const userSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+      maxLength: 22,
+    },
+    email: {
+      type: String,
+      required: true,
+      unique: true,
+      trim: true,
+      lowercase: true,
+      validate: [validator.isEmail, 'Provide valid Email Address'],
+    },
+    contact: {
+      type: Number,
+      maxLength: 12,
+      trim: true,
+      validate: [validator.isInt, 'Provide Numbers Only'],
```

Comment:
Instead of **Provide Numbers Only** write **Please provide a valid mobile number**